beets/mediafile.py

```diff
@@ -1643,6 +1643,18 @@ def update(self, dict):
         StorageStyle('COMPOSERSORT'),
         ASFStorageStyle('WM/Composersortorder'),
     )
+    work = MediaField(
+        MP3StorageStyle('TIT1'),
```

Comment:
Isn't this going to cause some issues since the same storage tag was previously used for a different tag?

Reply:
Yes that's exactly the point raised here: 

> There is a tag conflict between Work and Grouping: in picard, if you take the option "Save iTunes compatible grouping and work" (since Picard>=2.1.0) then the ID3v2 tag for grouping becomes GRP1 and the tag for work TIT1, if it isn't enabled then the tags are TIT1 and TXXX:WORK respectively. I chose the first one, but you might disagree. I feel like if we have to choose, let's be ITunes compatible.

I think I will just not touch the mediafile and revert the changes.